### PR TITLE
Create bang-the-dice-game.md

### PR DESCRIPTION
**Pull request by arnoudplantinga**

### CODE REVIEW

File: rulebooks/bang-the-dice-game.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+> or arrow, take it from the central pile. If you lose a life point or
+> arrow, return it to the pile.
+> You can exchange a three-bullet token for three one-bullet tokens
+> from the supply at any time (or vice-versa).
+
+# The Dice
+
+The dice show six different symbols. The symbols showing on the dice each have a different effect. You must apply all the dice results in the following order. You cannot pass on a rolled die: you must use them all!
+
+# 1. Indian arrow
```
Comment (camden) [inline rulebooks/bang-the-dice-game.md]: Can we use two `##` for these. This is so they appear as nested under "The Dice" in the table of contents.

File: rulebooks/bang-the-dice-game.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+---
+title: BANG! The Dice Game
+source: http://www.dvgiochi.net/bang_the_dice_game/BANG!_dice_game-rules.pdf
```
Comment (camden) [inline rulebooks/bang-the-dice-game.md]: Can you add the publisher?

Comment (arnoudplantinga) [reply rulebooks/bang-the-dice-game.md]: Done!

File: rulebooks/bang-the-dice-game.md (hunk 1 above)
Comment (arnoudplantinga) [reply rulebooks/bang-the-dice-game.md]: Good call, done!

**Pull request merged**